packages/sdk-metrics/test/aggregator/Histogram.test.ts

```diff
@@ -81,6 +81,27 @@ describe('HistogramAggregator', () => {
         sum: -65,
       });
     });
+
+    it('with single bucket', function () {
+      const aggregator = new HistogramAggregator([0], true);
```

Comment:
This should be a zero-length boundary.

```suggestion
      const aggregator = new HistogramAggregator([], true);
```

Reply:
Ah, yes, good catch. :+1: 
Thanks for pointing this out :slightly_smiling_face. Fixed in [1641bd7](https://github.com/open-telemetry/opentelemetry-js/pull/4456/commits/1641bd70d95b2c93c28ccc68d02abbcc42fe8716)